### DISCUSSION TITLE
Offline version of deepmd-kit, how to upgrade lammps?

### DISCUSSION THREAD

**Discussion by caolz** [deepmodeling/deepmd-kit#2702] [Q&A]
I have deepmd-kit-2.2.2-cpu-Linux-x86_64.sh installed, and the lammps version is 23 Jun 2022 - Update 1, and I'd like to use a version after 15Sep2022. If this is possible, how to do it?

**Comment by njzjz** [comment]
We only compile the stable version. If you want to use unstable versions, you may compile them by yourself.
